distributed/scheduler.py

```diff
@@ -2699,8 +2709,9 @@ def _transition_processing_erred(
         )
 
         for dts in ts.dependents:
-            dts.exception_blame = failing_ts
-            recommendations[dts.key] = "erred"
+            if not dts.who_has:
```

Comment:
isn't this the same as saying `for dts in ts.waiters`?

Reply:
Good point, looking at the code I think it _should_ mean the same. Let's see if CI complains.